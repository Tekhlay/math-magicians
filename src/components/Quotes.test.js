/* eslint-disable import/no-extraneous-dependencies */
import renderer from 'react-test-renderer';
import Quotes from './Qoutes';

describe('Test Quotes component', () => {
  it('Test component should render properly', () => {
    const tree = renderer.create(<Quotes />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
