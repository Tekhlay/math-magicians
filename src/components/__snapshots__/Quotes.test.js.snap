// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Quotes component Test component should render properly 1`] = `
<section
  className="qoute"
>
  <h1>
    "Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding."
  </h1>
  <span
    className="author"
  >
    - William Paul Thurston
  </span>
</section>
`;
